### PR TITLE
Change cm to cmu in sapling

### PR DESCRIPTION
**Pull request by oxarbitrage**
Part of https://github.com/zcash/zcash/issues/3446

For each of the 2 commits, suggested change was made, then fixed build errors until compile. Finally ran bitcoin and gtests, both of them passing.

### CODE REVIEW

File: src/zcbenchmarks.cpp [hunk 1]
```diff
@@ -596,7 +596,7 @@ double benchmark_create_sapling_spend()
     auto address = sk.default_address();
     SaplingNote note(address, GetRand(MAX_MONEY));
     SaplingMerkleTree tree;
-    auto maybe_cm = note.cm();
+    auto maybe_cm = note.cmu();
```
Comment (daira) [inline src/zcbenchmarks.cpp]: `s/maybe_cm/maybe_cmu/`

Comment (oxarbitrage) [reply src/zcbenchmarks.cpp]: Thanks @daira , changed at https://github.com/zcash/zcash/pull/4380/commits/8ebe538eaa67dcbf1299dc6cb7f8fb9945bc4272

**Comment by ebfull** [comment]
ACK

@zkbot r+

**Comment by daira** [comment]
ACK 8ebe538eaa67dcbf1299dc6cb7f8fb9945bc4272

**Pull request merged**